lib/__test__/rule-list.test.js

```diff
@@ -380,6 +386,31 @@ describe('rules', () => {
         ['x^-a / x^-b', 'x^(-a - -b)'],
     ])
 
+    suite('multiplying coefficients', rules.MULTIPLY_COEFFICIENTS, [
+        ['x^2 * y^2', '(1 * 1) (x^2 * y^2)'],
```

Comment:
Looking at the tests in collectAndCombineSearch.test.js mathsteps doesn't bother showing the `1` coefficients.
```
    ['x^2 * x * x',
      ['x^2 * x^1 * x^1',
        'x^(2 + 1 + 1)',
        'x^4'],
    ],
    ['y * y^3',
      ['y^1 * y^3',
        'y^(1 + 3)',
        'y^4'],
    ],
    ['2x * x^2 * 5x',
      ['(2 * 5) * (x * x^2 * x)',
        '10 * (x * x^2 * x)',
        '10x^4'],
      '10x^4'
    ],
```